specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -478,6 +478,122 @@
         }
       }
     },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/features/{featureName}/userpreference": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the user preferences for a given feature.",
+        "x-ms-examples": {
+          "addsServices_getUserPreference": {
+            "$ref": "./examples/UserPreference.json"
+          }
+        },
+        "operationId": "addsServices_getUserPreference",

```

Comment:
A recommended operation id would be `UserPreferences_Get` or `AddsServicesUserPreference_Get`
Also, just out of curiosity, what does `Adds` stand for

Reply:
@dsgouda, ok I will use AddsServicesUserPreference_get, since it aligns with the naming convention I have used for the previous operations also.
Adds is Active Directory Domain Service,

Reply:
This is done.